wasm2c/wasm-rt-impl.c

```diff
@@ -399,7 +348,7 @@ uint64_t wasm_rt_grow_memory(wasm_rt_memory_t* memory, uint64_t delta) {
 void wasm_rt_free_memory(wasm_rt_memory_t* memory) {
 #if WASM_RT_USE_MMAP
   const uint64_t mmap_size = get_allocation_size_for_mmap(memory);
-  os_munmap(memory->data, mmap_size); // ignore error
+  os_munmap(memory->data, mmap_size);  // ignore error
```

Comment:
Revert this line?

Reply:
SGTM

Reply:
BTW, IMHO, its generally advisable when creating a PR to run clang-format just on the lines that you touch (e.g. `git clang-format origin/main`), to avoid mixing real changes to formatting churn.  (i'm fine with leaving this line as part of this PR though since its just one line).